WASM.md

```diff
@@ -1,23 +1,68 @@
-Proof of Concept WebAssembly Support.
+WebAssembly Extension Support
 
-# WebAssembly VM
+# Build Dependencies
 
-The WebAssembly VM is https://github.com/WAVM/WAVM. This VM was chosen for the
-proof of concept because:
+## Install glib2.0 if required (on Debian/Ubuntu)
```

Comment:
Replace this line with dependency name alone (i.e. `glib2.0`).

Reply:
Done.